lib/server/wilson.js

```diff
@@ -118,9 +118,19 @@ module.exports = function(app, appWilsonConfigJson, dependencies) {
 
   //WILSON CLIENT
   router.get('/:version?/client.wilson.js', function(req, res) {
+    //Set max age headers based on config for wilson (NOTE: maxAgeMs is millis, so we divide to specify seconds)
```

Comment:
You might want to check if the req.params.version is set before setting the cache control. Cache should probably be max-age=0 for the case no version is provided.